ParkenDD/LotlistView/LotlistViewController.swift

```diff
@@ -76,6 +76,11 @@ class LotlistViewController: UITableViewController, UIViewControllerPreviewingDe
 
 		refreshControl?.endRefreshing()
 	}
+	
+	override func viewWillDisappear(_ animated: Bool) {
```

Comment:
This still doesn't fix the issue for me. However, you're on the right track. Try moving this to `SettingsViewController`'s `viewDidLoad`. Make sure that the title is the localized string.

Reply:
It seems to me that you would need it in both SettingsViewController and LotlistViewController. There seems to be some bug with the back navigation button when you go from LotlistVC to CityList and from SettingsVC to CityListVC. Just putting it only on SettingsVC does solve the problem.

Reply:
Provided a commit with a possible fix. Read comment above. Let me know.